detail/main.ipynb

```diff
@@ -113,20 +134,15 @@
     "\n",
     "tabs = pn.Tabs()\n",
     "tabs.extend([\n",
-    "    ('Structure', explorer.structure),\n",
-    "    ('Total energy', explorer.ener_plot),\n",
+    "    ('Process evaluation', explorer.process),\n",
+    "    ('Reported structure', explorer.structure_ref),\n",
+    "    ('DFT-optimization energy', explorer.ener_plot),\n",
+    "    ('DFT-optimized structure', explorer.structure_opt),\n",
```

Comment:
could you comment one of the structures for the moment in order not to break things?